packages/reports/src/Components/Routes/__tests__/binary-link.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import userEvent from '@testing-library/user-event';
+import { BinaryLink } from '../index';
+import { findRouteByPath, normalizePath } from '../helpers';
+
+jest.mock('../helpers', () => ({
+    findRouteByPath: jest.fn(),
+    normalizePath: jest.fn(),
+}));
+
+const MockBinaryLink = ({ to }: { to: string }) => (
+    <BrowserRouter>
+        <BinaryLink active_class='active_class' to={to}>
+            <div data-testid='dt_child' />
+        </BinaryLink>
+    </BrowserRouter>
+);
+
+describe('<BinaryLink />', () => {
+    beforeEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should render "children" when passed in', () => {
+        (normalizePath as jest.Mock).mockReturnValue('/test-route');
+        const mockRoute = {
+            path: '/test-route',
+            exact: true,
+        };
+        (findRouteByPath as jest.Mock).mockReturnValue(mockRoute);
+        render(<MockBinaryLink to='/test-route' />);
+        expect(screen.getByTestId('dt_child')).toBeInTheDocument();
+    });
+    it('should have "active_class" when passed in', () => {
+        const mockRoute = {
+            path: '/test-route',
+            exact: true,
+        };
+        (findRouteByPath as jest.Mock).mockReturnValue(mockRoute);
+        (normalizePath as jest.Mock).mockReturnValue('/test-route');
+
+        render(<MockBinaryLink to='/test-route' />);
+        userEvent.click(screen.getByTestId('dt_binary_link'));
+        const link = screen.getByTestId('dt_binary_link');
+        expect(link).toHaveClass('active_class');
+    });
+    it('should render "NavLink" when valid "to" property is passed', () => {
+        const mockRoute = {
```

Comment:
We are declaring mockRoute 3 times, maybe it'll make sense to declare it once and reuse?